ztrouy/house-rules
# Feature - Home View

### Purpose
To create a default view for logged in Users

### Files Changed
- Imported Bootstrap, updated application title, and removed the Vite favicon in `index.html`
- Deleted the Vite icon `react.svg`
- Updated the port used for the proxy in `vite.config.js`
- Commented out default template css in `index.css` and `App.css`
- Added inline stying to recenter text in `Login.jsx` and `Register.jsx`
- Created file and added rendering for Home view in new `Home.jsx` component module
- Added routing for Home view, and removed accidentally copied Routes and imports from previous project in `ApplicationViews.jsx`
- Added "AutoMapper" to list of technologies used in `README.md`

### To Test
Fetch, setup and run according to project README, then confirm the following
- User is automatically redirected to login page when not logged in
- Default view when logged in is the Home view

closes #3
